File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        dict = {}
        sum = 0
        for ele in nums:
            dict[ele] = dict.get(ele,0)+1
        for key in dict.keys():
            if dict[key]==1:
                sum+=key
        return sum

File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
       HashMap<Integer, Integer>dict = new HashMap<Integer, Integer>();
		for(int num : nums){
		    if (dict.containsKey(num)==true)
		        dict.put(num, dict.get(num)+1);
		    else
		        dict.put(num, 1);
		}
		
		int output = 0;
		for(int i : dict.keySet()){
		    if(dict.get(i) == 1)
		        output += i;
		}
		 
		return output; 
    }
}